### PR TITLE
Fixing e2e test case ids and failing tests on 3.19

### PR DESCRIPTION
**Pull request by yellowee**
Backporting https://github.com/saleor/saleor-dashboard/pull/4764 and fixing failing tests

## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [ ] 🛠 Workflow CI/CD changes

## Related Issues or Documents

- closes #

## Have you written tests?
- [ ] Yes!
- [ ] No... here is why: _Writing tests are mandatory, please replace this text with why test are not included in this PR_

### CODE REVIEW

**Review by szczecha — APPROVED** [review]

File: .changeset/serious-books-wave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
```
Comment (poulch) [inline .changeset/serious-books-wave.md:2]: suggestion: imo this should be patch

File: .changeset/serious-books-wave.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+Changed e2e test case ids to remove duplicate case ids and fix inconsistencies between repo and Testmo. Thanks to this, our QA reports will be more accurate
```
Comment (poulch) [inline .changeset/serious-books-wave.md:5]: I don't see here any changes related with ids

File: .changeset/serious-books-wave.md (hunk 1 above)
Comment (yellowee) [reply .changeset/serious-books-wave.md]: fixed

**Review by Cloud11PL — APPROVED** [review]

File: .changeset/serious-books-wave.md (hunk 2 above)
Comment (yellowee) [reply .changeset/serious-books-wave.md]: You're right - the changes were needed in Testmo, I've corrected the changeset description

**Review by poulch — APPROVED** [review]

**Review by michalina-graczyk — APPROVED** [review]

**Pull request merged**